common/crypto_utils/avalon_crypto_utils/crypto_utility.py

```diff
@@ -37,7 +41,7 @@ def generate_iv():
     Function to generate random initialization vector
     """
 
-    return crypto.SKENC_GenerateIV()
+    return get_random_bytes(12)
```

Comment:
Please add a comment what does 12 mean. Use constant variables

Reply:
Mention that 96 bits of randomness is recommended to prevent birthday attacks (duplicate IVs).